src/bluesky/tests/test_new_examples.py

```diff
@@ -285,7 +304,8 @@ def test_mvr_with_timeout(hw):
     actual = list(mvr(hw.motor1, 1, hw.motor2, 2, timeout=42))
     for msg in actual[:2]:
         msg.command == "set"  # noqa: B015
```

Comment:
Might as well fix this test while here!

```suggestion
        assert msg.command == "set"  # noqa: B015
```